documentation/book/assembly-configuring-kafka-listeners.adoc

```diff
@@ -0,0 +1,23 @@
+// This assembly is included in the following assemblies:
+//
+// cluster-operator.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {context}
+
+[id='assembly-configuring-kafka-broker-listeners-{context}']
+= Configuring Kafka broker listeners
+
+{ProductName} allows users to configure the listeners which will be enabled in Kafka brokers.
+Two types of listeners are supported:
+
+* Plain listener on port 9092 (without encryption)
+* TLS interface on port 9093 (with encryption)
```

Comment:
Why "interface" and not "listener" as the previous one?

Reply:
Should be probably listener. You are right. I will fix it.